packages/client/src/bags/types/bag.types.ts

```diff
@@ -30,4 +30,5 @@ export type Bag = {
   '@controls'?: {
     [key: string]: HypermediaLink;
   };
+  promocodes: string[];
```

Comment:
Question: Is promocodes always available on the payload? If the bag does not have a promocode applied, will it return an empty array?

Reply:
That's correct. It will return an empty array if a promocode has not been applied.

Reply:
Ok, thank you!